mypy/test/teststubgen.py

```diff
@@ -778,6 +780,20 @@ def test(arg0: str) -> None:
         assert_equal(output, ['def test(arg0: str) -> Action: ...'])
         assert_equal(imports, [])
 
+    def test_generate_c_property_with_pybind11(self) -> None:
+        """Signatures included by PyBind11 inside property.fget are read."""
+        class TestClass:
+            def get_attribute(self) -> str:
```

Comment:
Now the type seems to be also available through the return type here. Maybe leave out the return type to make sure that we can figure out the property type from the docstring only? Or use a different return type in the annotation to check that docstring takes precedence.

Reply:
Good point!